LoopKitUI/Views/DemoPlaceHolderView.swift

```diff
@@ -17,7 +17,7 @@ public struct DemoPlaceHolderView: View {
     
     public var body: some View {
         ZStack {
-            Color.accentColor.opacity(0.1).ignoresSafeArea()
+            Color(frameworkColor: "demo-background").ignoresSafeArea()
```

Comment:
This strategy does not allow Workspace level overriding, like the accent/insulin/glucose colors.  This is a very "Tidepool" brand color. Do we want DIY using the same color?

Reply:
For now I think the answer is yes, since the image on that placeholder screen is also using Tidepool colours (and we don't have a DIY alternative).

Reply:
Color scheme is one of the main things that visually identifies Tidepool Loop apart from DIY Loop. I don't think we want this in DIY.

Reply:
And yeah your point about the image being very "tidepooolesque" is also a concern. We need to pick non-shared places to put Tidepool distinguishing UI, or build ways to make that content switchable at the LoopWorkspace level.

Reply:
@ps2 yes, I agree. But this isn't the right ticket for that work. This is just about getting a demo version ready to be used. Shall I move forward with this? Or do we need to solve this at this time?

Reply:
Merges to DIY have been happening frequently, and the other code from LOOP-4640 is already in dev and will likely be merged into DIY soon. We need to figure out something sooner than later; I've opened up discussions in Slack and on the Jira ticket.